### PR TITLE
Update es_mx.json

### PR DESCRIPTION
**Pull request by cerealconjugo**
- added 1.20's missed items strings.
- Uncapitalized a lot of things for language consistency.
- some corrections here and there.

### CODE REVIEW

File: src/main/resources/assets/farmersdelight/lang/es_mx.json [hunk 1]
```diff
@@ -279,9 +299,12 @@
 
   "farmersdelight.jei.cooking": "Cocinando",
   "farmersdelight.jei.cutting": "Tabla de picar",
+  "farmersdelight.jei.chance": "%1$s%% prbabilidad",
```
Comment (vectorwing) [inline src/main/resources/assets/farmersdelight/lang/es_mx.json]: There's a small typo here. Should likely be "probabilidad".

File: src/main/resources/assets/farmersdelight/lang/es_mx.json [hunk 2]
```diff
@@ -279,9 +299,12 @@
...
   "farmersdelight.jei.decomposition": "Descomposición",
   "farmersdelight.jei.decomposition.light": "Acelerado por la luz del sol",
   "farmersdelight.jei.decomposition.fluid": "Acelerado por el agua adyacente",
   "farmersdelight.jei.decomposition.accelerators": "Acelerado por activadores adyacentes (ver más abajo)",
-  "farmersdelight.jei.info.straw": "Una fibra vegetal seca utilizada en la artesanía. Se obtiene cortando cultivos y plantas herbáceas con un Cuchillo."
+  "farmersdelight.jei.info.straw": "Una fibra vegetal seca utilizada en la artesanía. Se obtiene cortando cultivos y plantas herbáceas con un Cuchillo.",
+  "farmersdelight.jei.info.knife": "Los cuchillos son armas de comabate cuerpo a cuerpo ligeras. Con ellos puedes recolectar paja de los pastos, además de que garantiza obtener los drops secundarios de los animales.",
```
Comment (vectorwing) [inline src/main/resources/assets/farmersdelight/lang/es_mx.json]: "Comabate" seems like a typo as well.

**Comment by vectorwing** [comment]
Thanks for the update! I left comments on two typos, but otherwise, everything seems good. 👍

File: src/main/resources/assets/farmersdelight/lang/es_mx.json (hunk 2 above)
Comment (cerealconjugo) [reply src/main/resources/assets/farmersdelight/lang/es_mx.json]: Fixed!🫡

**Pull request merged**